lambda/src/main/scala/com/gu/meeting/ChatMessage.scala

```diff
@@ -0,0 +1,70 @@
+package com.gu.meeting
+
+import com.gu.meeting.ChatMessage.logger
+import com.typesafe.scalalogging.StrictLogging
+import io.circe.Encoder
+
+import java.time.OffsetDateTime
+
+object ChatMessage extends StrictLogging {
+
+  def chatMessagesForEvents(
+      events: List[MeetingData],
+      minuteOfInterest: OffsetDateTime,
+  ): List[(String, ChatMessage)] =
+    groupAndFilterAllEvents(events, minuteOfInterest).flatMap(chatMessage)
+
+  def groupAndFilterAllEvents(
+      events: List[MeetingData],
+      minuteOfInterest: OffsetDateTime,
+  ): List[List[MeetingData]] = {
+    val instantOfInterest = minuteOfInterest.toInstant
+    val endOfDay = minuteOfInterest.withHour(0).plusDays(1).toInstant
+    val eventsByCalendar =
+      events
+        .filter(_.creator.exists(_.endsWith("@guardian.co.uk")))
+        .groupBy(_.organiser.map(_._1))
```

Comment:
it groups it by organiser (owning calendar) but it doesn't actually check to see if the web hook link exists or is the same.